### PR TITLE
Initalize go modules, cleanup .travis.yml

### PR DESCRIPTION
**Pull request by Gonzih**
By switching to go modules it is possible to simplify CI pipeline and make sure all dependencies are versioned and verified via go.sum file.

You also dont need to keep your code in GOPATH with modules on.

But you will have to use go version 1.11 for them to work well though (since 1.12 just came out, it should not be a problem I think).

### CODE REVIEW

**Comment by Gonzih** [comment]
I struggle to figure out why CI is failing, please wait for me to fix this, might take couple of days though.

**Comment by Gonzih** [comment]
Seems like go specific details of the build are fixed, there are some missing libs still though. Any tips on what am I doing wrong are welcome :)

File: .travis.yml [hunk 1]
```diff
@@ -2,42 +2,31 @@ dist: xenial
...
   - sudo apt-get -qq update
   - sudo apt-get install -y  limesuite libaec-dev libaec0 liblimesuite-dev libcorrect libsathelper libsoapysdr0.6 libairspy0 libsoapysdr-dev libairspy-dev git g++ cmake libsqlite3-dev libi2c-dev libusb-1.0-0-dev
-  - echo "Building RTLSDR"
-  - git clone https://github.com/librtlsdr/librtlsdr.git
-  - cd librtlsdr
-  - mkdir build && cd build
-  - cmake ..
-  - make -j10
-  - sudo make install
-  - sudo ldconfig
```
Comment (racerxdl) [inline .travis.yml]: You removed this build process, and this is nescessary for building it.

**Comment by racerxdl** [comment]
I fixed the build deps on master, added stable limesuite  to fix the LMS issues.

**Review by racerxdl — CHANGES_REQUESTED** [review]
The master branch is now building, the librtlsdr and libLimeSuite should be built before building SatHelperApp.

File: .travis.yml (hunk 1 above)
Comment (Gonzih) [reply .travis.yml]: Ok, i think I understand this flow now. I thought that scripts in ./travis-build were just a duplication, but they are for release.

**Comment by Gonzih** [comment]
@racerxdl fixed!

**Review by racerxdl — APPROVED** [review]

**Comment by racerxdl** [comment]
Great! Now I need to delete my local repository due camelCase XD

**Comment by Gonzih** [comment]
yeah, based on my observation camelCase would work if your org would use camel case. For example my nickname is capitalized and it works with most of go tooling. But I saw issues related to this in the past. I remember logrus author had to change his account name because it was breaking a lot of go tooling. Now both versions are supported due to github redirect mechanism. So you get get it via `https://github.com/sirupsen/logrus` or `https://github.com/Sirupsen/logrus`. Rule of thumb is to prefer lowercase. In import paths and package names.

Thanks for merging this. I will try to dig deeper in to this project to see where I can help :)

**Comment by racerxdl** [comment]
Sure, I need to thank you for the changes. I havent checked go modules yet, looks better than go vendor (at least I think its better than commiting 2GB of deps inside the repository).

Thanks, and let me know if you need anything!

We also have a RocketChat to talk about other projects https://osp.teske.net.br/
And a Telegram Channel: https://t.me/OpenSatelliteProject

Feel free to join if you want.

**Pull request merged**